charts/fluentd/README.md

```diff
@@ -90,6 +90,9 @@ helm install -n monitoring \
 | `daemonset.logzioRetryForever` | If true, plugin will retry flushing forever | `true` |
 | `daemonset.logzioFlushThreadCount` | Number of threads to flush the buffer. | `2` |
 | `daemonset.logzioLogLevel` | The log level for this container. | `info` |
+| `daemonset.excludeFluentdPath` | Path to fluentd logs, to exclude them from the logs that Fluent tails. | `/var/log/containers/*fluentd*.log` |
+| `daemonset.extraExclude` | More paths to exclude from the Fluentd source that tails containers logs. | `""` |
```

Comment:
is it single value or multiple? if multiple can you explain more and give example?